### ISSUE TITLE
Android andronix & debian illegal instruction

### ISSUE THREAD

**Issue by iinta** [SpatiumPortae/portal#147]
I got Motorola snapdragon with andronix debian( Linux)

Used your guide to get latest version but it says ' illegal instruction" I guess it's cos CPU.  At compile should Include arm? Arc.

I'm no sw expert to compile this by myself and the linuxbrew denied co/operation would do it by myself..

**Comment by ZinoKader** [comment]
The releases are targeting a lot of architectures. Have you tried this specific debian package?

https://github.com/SpatiumPortae/portal/releases/download/v1.2.3/portal_1.2.3_Linux_arm64.deb

**Comment by ZinoKader** [comment]
> [Detected OS: linux_amd64]

It seems like the OS detection is not doing a great job of detecting your OS, as you say you're running Debian. As you can see, the file fetched is and installed is the following

https://github.com/SpatiumPortae/portal/releases/download/v1.2.3/portal_1.2.3_Linux_x86_64.tar.gz

What I suggest for you to get a working version going is to use this binary instead, as it is the correct one for your OS: https://github.com/SpatiumPortae/portal/releases/download/v1.2.3/portal_1.2.3_Linux_arm64.deb

**Comment by iinta** [comment]
Yes though so that the CPU architecture was the problem because the error message.
. Just followed the instructions, it is modified version from Google play.
Thanks for the arm link trying that.
